Diana_Mokhnitskaya/Lesson_5/HW5_task3.py

```diff
@@ -0,0 +1,11 @@
+import re
+import json
+
+task_string = " name=Amanda=sssss&age=32&&salary=1500&currency=euro "
+split_task_string = str(re.split('=|&|sssss', task_string))
+print("The list after performing split functionality : " + str(split_task_string))
+for values in split_task_string:
+    str(values).replace(' ', '')
+    print(json.loads(split_task_string))
+
+# Я не понимаю, как правильно реализовать переход от строки в словарь
```

Comment:
'''
task_string = " name=Amanda=sssss&age=32&&salary=1500&currency=euro "
filtered = task_string.strip()
filtered = filtered.replace('&&', '&').replace('&', '=').replace('sssss', '').replace('==', '=')
print(filtered)
result = filtered.split('=')
print(result)
dict_info = dict()
for value in range(0, len(result)):
    if (value % 2) != 0:
        continue
    dict_info[result[value]] = result[value + 1]
'''

it is quick solution. You can modify it